Makefile.toml

```diff
@@ -527,7 +527,9 @@ script = '''
 #!/usr/bin/env bash
 
 tmux list-windows -t risedev -F "#{pane_id}" | xargs -I {} tmux send-keys -t {} C-c C-d
-tmux kill-session -t risedev
+tmux kill-session -t risedev &
+kill_session_pid=$!
```

Comment:
Wait... Can this work? I suppose `$!` here refer to `tmux kill-session` command, which must have certainly exited. The problem is whether all RisingWave processes have exited. We need to wait for RisingWave processes rather than `tmux kill-session` command.

Reply:
Looks really not work, the test on pr was just accidental.
https://buildkite.com/risingwavelabs/main/builds/5218

Reply:
Any ideas on that? sleep?

Reply:
u can also just loop through and grep on the output of `list-windows`. If empty, I guess it means all have exited.